src/opnsense/mvc/app/views/OPNsense/IPsec/sessions.volt

```diff
@@ -116,6 +116,15 @@
         });
 
         updateServiceControlUI('ipsec');
+
+        /* for ipsec tunnels widget */
+        $('#grid-phase1').on("loaded.rs.jquery.bootgrid", function () {
+            handleSearchAndEdit('#grid-phase1');
+        });
+        $(window).on('hashchange', function () {
```

Comment:
since we always seem to handle this case after the event registration, can this not be moved into `handleSearchAndEdit`? The identifier is the same